src/plugins/interactive-media/interactive-media-info.js

```diff
@@ -48,15 +49,37 @@ class InteractiveMediaInfo {
     return {
       sourceType: MEDIA_SOURCE_TYPE.internal,
       sourceUrl: '',
-      playbackRange: [0, 1],
       copyrightNotice: '',
+      playbackRange: [0, 1],
       width: 100,
       aspectRatio: MEDIA_ASPECT_RATIO.sixteenToNine,
       showVideo: false,
       chapters: [this.getDefaultChapter(t)]
     };
   }
 
+  validateContent(content) {
+    const schema = joi.object({
+      sourceType: joi.string().valid(...Object.values(MEDIA_SOURCE_TYPE)).required(),
+      sourceUrl: joi.string().allow('').required(),
+      copyrightNotice: joi.string().allow('').required(),
+      playbackRange: joi.array().items(joi.number().min(0).max(1)).required(),
+      width: joi.number().min(0).max(100).required(),
+      aspectRatio: joi.string().valid(...Object.values(MEDIA_ASPECT_RATIO)).required(),
+      showVideo: joi.boolean().required(),
+      chapters: joi.array().items(joi.object({
+        key: joi.string().required(),
+        startPosition: joi.number().min(0).max(1).required(),
+        title: joi.string().allow('').required(),
+        text: joi.string().allow('').required(),
+        answers: joi.array().items(joi.string().allow('')).required(),
+        correctAnswerIndex: joi.number().min(-1).required()
+      })).required()
+    });
+
+    joi.attempt(content, schema, { convert: false, noDefaults: true });
+  }
```

Comment:
Really good that you added these plugin validation! 👏

Reply:
Yeah, it brought a few things into daylight. From now on we can be more confident that we do not save broken data.

Reply:
You know what we could do as an ultimate test of these changes?
Run a full import from ELMU into OMA on Staging after this ticket is brought to both.
Then if there are any inconsistencies, the import report will point them out.